### ISSUE TITLE
JAWs + Edge: Doesn`t Support Checkbox in Table

### ISSUE THREAD

**Issue by Lijing29** [OfficeDev/office-ui-fabric-react#10185]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)
- **Screen reader(s)**: (fill this out if applicable)

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/lijinglin29/pen/NWKrmyQ

#### Actual behavior:
![image](https://user-images.githubusercontent.com/50400285/63245836-b4c06700-c293-11e9-8b71-2d34771d8d28.png)
The first checkbox is created by Input Tag; The second checkbox is created by the Checkbox component provided by Fabric UI Component

1. Open codepen using Edge
2. Open JAWs
3. Use up/down arrow to navigate the table
4. When navigate to the first checkbox cell, it would read "Row 1 Include Fields Column 1 Year Checkbox not check Year" [Right]
4. Click down arrow, it would read "Average Category Checkbox not check" [Wrong]
5. Click down arrow, it would read "Row 3 Include Fields Column 1 Blank" [Wrong]
6. Click down arrow, it would read "Blank" [Wrong]
7. Click down arrow, it would read "Summarize value by column 2 Not a value" [Right]

#### Expected behavior:

Read the cell contains Checkbox component as the same with the cell containing checkbox created by Input Tag

**Comment by kenotron** [comment]
This reads okay on Narrator + Edge which is our platform that we support: 
https://github.com/OfficeDev/office-ui-fabric-react/wiki/Accessibility-Troubleshooting

The bug is likely external to JAWS

**Comment by Lijing29** [comment]
JAWs + IE works well on the first checkbox which is created by input tag. However, it doesn`t work on the second Checkbox which is created by Fabric UI Component. It seems it is an issue in Fabric UI Component.
